modules/021-cni-cilium/monitoring/grafana-dashboards/kubernetes-cluster/cilium-agent.json

```diff
@@ -8582,7 +8582,7 @@
     ]
   },
   "timezone": "browser",
-  "title": "Cilium v1.12 Agent Metrics",
+  "title": "Cilium Agent Metrics",
```

Comment:
The main problem is not naming, we cannot update dashboard when update to cilium 1.14.